pennylane/transforms/mitigate.py

```diff
@@ -528,6 +528,11 @@ def circuit(w1, w2):
 
     def processing_fn(results):
         """Maps from input tape executions to an error-mitigated estimate"""
+        if qml.active_return():
+            for i, tape in enumerate(out_tapes):
+                # stack the results if there are multiple measurements
+                if len(tape.observables) > 1:
+                    results[i] = qml.math.stack(results[i])
```

Comment:
No deprecation warning is raised here, correct?

Reply:
No deprecation warning is raised here.

Reply:
No, just inquired if NumPy's ragged array warning was raised here.

Reply:
Ah sorry I misunderstood. Just checked and there are no NumPy ragged array deprecation warnings 👍 .

Reply:
I don't know if ZNE supports other measurements than expval, but if it is the case then it will create ragged arrays.

Reply:
Confirmed by @Qottmann, only support expval!